jetty-websocket/websocket-javax-common/src/main/java/org/eclipse/jetty/websocket/javax/common/messages/AbstractDecodedMessageSink.java

```diff
@@ -32,12 +32,12 @@ public abstract class AbstractDecodedMessageSink implements MessageSink
 {
     private static final Logger LOG = LoggerFactory.getLogger(AbstractDecodedMessageSink.class);
 
-    private final MethodHandle _methodHandle;
+    private final MethodHolder _methodHolder;
     private final MessageSink _messageSink;
 
-    public AbstractDecodedMessageSink(CoreSession coreSession, MethodHandle methodHandle)
+    public AbstractDecodedMessageSink(CoreSession coreSession, MethodHolder methodHolder)
```

Comment:
Signature change of public class. Internal or keep previous signature deprecated?